Install-Kite.md

```diff
@@ -11,3 +11,19 @@ chmod +x {{site.dataset-command}}
 ```
 
 The first command downloads the Kite tools JAR to a local file named _{{site.dataset-command}}_. The second changes the access rights on the JAR so that the classes in the JAR are executable.
+
+Some versions of the Cloudera QuickStart VM comes with Kite installed. However, it might not be the latest version. To check the version, from a terminal window, enter the following command.
+
+```
+kite-dataset --version
+```
+
+If the version number doesn't match the version you downloaded, you have two options.
+
+1. Include the path to `kite-dataset` when you invoke commands. For example, if the JAR file is in the current directory, use `./kite-dataset`.
+
+1. Replace the installed version of `kite-dataset.jar`.
```

Comment:
No need to use the `jar` file extension, this is usually called `kite-dataset`. Also, this is `{{site.dataset-command}}` above, should that be used here as well?